pom.xml

```diff
@@ -70,17 +47,12 @@
                 <artifactId>liberty-maven-plugin</artifactId>
                 <version>${version.liberty-maven-plugin}</version>
                 <configuration>
-                    <install> <!-- Using Daily Builds until ee-11 features are in beta -->
-                        <runtimeUrl>https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/nightly/2024-03-25_0300/openliberty-all-24.0.0.4-cl240420240325-0300.zip</runtimeUrl>
-                    </install>
-                    <!-- re-enable when concurrency 3.1 is in beta
```

Comment:
This is switching the GA build instead of the nightly, since the GA build has Concurrency 3.0